platforms/access-bridge/src/global.d.ts

```diff
@@ -0,0 +1,13 @@
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv {
+      API_SECRET: string;
+      STRIPE_SECRET: string;
+      BIND_ADDR: string;
+      BIND_PORT: string;
+      ACCESS_CONTROL_CLIENT: string;
+      SIMS_CLIENT: string;
```

Comment:
These seem like hosts, not clients, right?
Naming tweak to be more meaningful than our internal team names.
```suggestion
      PASSPORT_API_HOST: string;
      SUBSCRIPTION_API_HOST: string;
```

Reply:
The Subscription doesn't really make anymore sense, since we are using external providers and not internal anymore. So, I did name it SIMS_API_HOST.